quickstep/src/com/android/quickstep/RecentsAnimationDeviceState.java

```diff
@@ -95,8 +95,7 @@ public class RecentsAnimationDeviceState implements DisplayInfoChangeListener {
     private final RotationTouchHelper mRotationTouchHelper;
     private final TaskStackChangeListener mPipListener;
     // Cache for better performance since it doesn't change at runtime.
-    private final boolean mCanImeRenderGesturalNavButtons =
-            InputMethodService.canImeRenderGesturalNavButtons();
+    private final boolean mCanImeRenderGesturalNavButtons = isImeRenderingNavButtons();
```

Comment:
`isImeRenderingNavButtons` doesn't look like it can substitute `InputMethodService.canImeRenderGesturalNavButtons`.
I'd keep the call on A13 and fallback to a hardcoded value for older versions.